server/vcs/provider/github/converter/pr_review_event.go

```diff
@@ -1,36 +1,63 @@
 package converter
 
 import (
+	"fmt"
 	"github.com/google/go-github/v45/github"
 	"github.com/palantir/go-githubapp/githubapp"
 	"github.com/pkg/errors"
 	"github.com/runatlantis/atlantis/server/events/models"
 	"github.com/runatlantis/atlantis/server/neptune/gateway/event"
+	"time"
 )
 
 type PullRequestReviewEvent struct {
 	RepoConverter RepoConverter
+	PullConverter PullConverter
 }
 
 func (p PullRequestReviewEvent) Convert(e *github.PullRequestReviewEvent) (event.PullRequestReview, error) {
 	installationToken := githubapp.GetInstallationIDFromEvent(e)
+	if e.GetRepo() == nil {
+		return event.PullRequestReview{}, fmt.Errorf("repo is nil")
+	}
 	repo, err := p.RepoConverter.Convert(e.GetRepo())
 	if err != nil {
 		return event.PullRequestReview{}, errors.Wrap(err, "converting repo")
 	}
+
+	if e.GetPullRequest() == nil {
+		return event.PullRequestReview{}, fmt.Errorf("pull request is nil")
+	}
+	pull, err := p.PullConverter.Convert(e.GetPullRequest())
+	if err != nil {
+		return event.PullRequestReview{}, errors.Wrap(err, "converting pull request")
+	}
+
+	if e.GetSender() == nil {
```

Comment:
Should we really be erroring out if any of these are nil? Like what are we supposed to do if GH is missing metadata.  Can't we just gracefully handle these like we do for other events? All these Get methods already should have nil checks there so it's safe to access underlying members since we'll just get the zero value of those.

Reply:
I was thinking we should go back and add the nil checks to all of the other events missing them because they don't return the zero values, they return nil or a pointer.